app/models/pillar.rb

```diff
@@ -41,7 +41,8 @@ def all_pillars
         ldap_admin_group_name:   "ldap:admin_group_name",
         ldap_tls_method:         "ldap:tls_method",
         ldap_mail_attribute:     "ldap:mail_attribute",
-        cloud_framework:         "cloud:framework"
+        cloud_framework:         "cloud:framework",
+        insecure_registries:     "docker:registries"
```

Comment:
Are we gonna use `insecure_registries`? Because we'll handle insecure, secure and mirrors.

Reply:
yeah, this was the docker term originally for the command line flag, but it's outdated. i think we can change it now

Reply:
👍